snapcraft/internal/sources.py

```diff
@@ -389,7 +389,13 @@ def get(sourcedir, builddir, options):
     handler_class = _get_source_handler(source_type, options.source)
     handler = handler_class(options.source, sourcedir, source_tag,
                             source_branch)
-    handler.pull()
+    try:
+        handler.pull()
+    except FileNotFoundError:
```

Comment:
That's what I got when I tried it out.

On Thu, Aug 18, 2016 at 4:43 PM, Sergio Schvezov <notifications@github.com>
wrote:

> In snapcraft/internal/sources.py
> <https://github.com/snapcore/snapcraft/pull/742#discussion_r75385497>:
>
> > @@ -389,7 +389,13 @@ def get(sourcedir, builddir, options):
> >      handler_class = _get_source_handler(source_type, options.source)
> >      handler = handler_class(options.source, sourcedir, source_tag,
> >                              source_branch)
> > -    handler.pull()
> > +    try:
> > +        handler.pull()
> > +    except FileNotFoundError:
>
> is this really a FileNotFoundError?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/snapcore/snapcraft/pull/742/files/140fe4bd0e097919628da4137a883b67976bb0e7#r75385497>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAnbyHk6LxaVfAUW31BHlxJgYhmiLUsaks5qhMQDgaJpZM4Jn4qH>
> .
>